lbz/request.py

```diff
@@ -84,10 +103,48 @@ def json_body(self) -> dict | None:
                 raise BadRequestError(f"Content-Type header is missing or wrong: {content_type}")
         return self._json_body
 
-    def to_dict(self) -> dict:
-        copied = {k: v for k, v in self.__dict__.items() if not k.startswith("_")}
-        copied["headers"] = dict(copied["headers"])
-        copied["user"] = repr(copied["user"])
-        if copied["query_params"] is not None:
-            copied["query_params"] = dict(copied["query_params"])
-        return copied
+
+class WebSocketRequest(BaseRequest):
+    """Represents request from Web Socket Secure API Gateway."""
+
+    def __init__(
+        # pylint: disable=too-many-positional-arguments
+        self,
+        body: str | bytes | dict,
+        request_details: dict,
+        context: dict,
+        is_base64_encoded: bool,
+        user: User | None = None,
+        headers: CIMultiDict | None = None,
+    ):
+        self.headers = headers
+        self.context = context
+        self.user = user
+        self.action = request_details.pop("routeKey")
```

Comment:
It would be nice to confirm the documentation that you used as I see that all these keys are under `context` 🤷

Reply:
I've run the code ;) I can share the actual payloads - they will be there in future PRs (around Event)

Reply:
Ok, so, I will verify it then 👍

Reply:
FYI: #105